bootstrap/README.md

```diff
@@ -1,42 +1,49 @@
 # Bootstrapping cabal-install
 
-This utility is only intended for use in building cabal-install
-on a new platform. If you already have a functional (if dated) cabal-install
-please rather run `cabal v2-install`.
+This utility is only intended for use in building `cabal-install`
+on a new platform. If you already have a functional (if dated) `cabal-install`,
+please, rather run `cabal install cabal-install`.
```

Comment:
```suggestion
on a new platform. If you already have a functional cabal-install, however old, then
please instead run `cabal install cabal-install`.
```

Reply:
Ah, I'd not been aware of that.

Reply:
Right. Only they have a little junky UI. For example, you are not supposed to actually merge them -- that's the job of the PR author. You are supposed to create them. In this case, I created them because that seemed most expedite to implement your suggestions. But the author creating suggestions is a very unusual workflow :-)